File: app/src/main/java/com/tournamentbracketjava/Network/FinalsGridApi.java
package com.tournamentbracketjava.Network;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class FinalsGridApi {

    private static Retrofit retrofit;
    private static final String Api_Url = "";

    public static Retrofit getFinalsGrid() {
        if (retrofit == null) {
            retrofit = new retrofit2.Retrofit.Builder()
                    .baseUrl(Api_Url)
                    .addConverterFactory(GsonConverterFactory.create())
                    .build();
        }
        return retrofit;
    }

}

File: app/src/main/res/layout/com/bracket/Network/FinalsGridApi.kt
package com.bracket.Network

import retrofit2.converter.gson.GsonConverterFactory
import retrofit2.Retrofit

class FinalsGridApi{

    companion object {

        private var retrofit: Retrofit ?=null
        private const val Api_Url: String = ""

        fun getFinalsGrid(): Retrofit? {
            if (retrofit == null) {
                retrofit = retrofit2.Retrofit.Builder()
                    .baseUrl(Api_Url)
                    .addConverterFactory(GsonConverterFactory.create())
                    .build()
            }
            return retrofit
        }
    }

}